frontend/src/App.css

```diff
@@ -50,7 +50,7 @@
 
 .BackGroundImage {
   background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
-    url('./backgroundImg.jpeg');
+    url('./backgroundImg.jpeg'); /*Image credits: https://unsplash.com/*/
```

Comment:
Nice find Sree. The artist wasn't credited before which would have been unfair to them.